project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
+patch release;
+
+After each release (both "major" (vX.Y.0) and "patch" releases (vX.Y.Z)), a
+patch-release milestone (vX.Y.Z + 1) is created.
```

Comment:
Yes, I thought about that for a bit, I wasn't really happy with "next" because that could either be seen as "next major" or "next minor/patch" release.

I agree that adding something to the milestone name to better reflect the intent would be good.

Perhaps `vX.Y.Z+1-candidates`? (`vX.Y.Z+1-maybe` works, but feels a bit "yeah, well, maybe, idk"). Open to more/better suggestions